space/space.go

```diff
@@ -302,7 +302,7 @@ func (r *GormRepository) LoadByOwner(ctx context.Context, userID *uuid.UUID, sta
 
 func (r *GormRepository) LoadByOwnerAndName(ctx context.Context, userID *uuid.UUID, spaceName *string) (*Space, error) {
 	res := Space{}
-	tx := r.db.Where("spaces.owner_id=? AND spaces.name=?", *userID, *spaceName).First(&res)
+	tx := r.db.Where("spaces.owner_id=? AND LOWER(spaces.name) LIKE ?", *userID, "%"+strings.ToLower(*spaceName)+"%").First(&res)
```

Comment:
The original issue was only referring to case sensitive match (#1275), but here it goes beyond that check for any name that includes the given substring.